starter-code/index.html

```diff
@@ -1,31 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
-<body>
-  Instagram
-
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
+<body>
+  <div class="background">
+    <div class="container">
+      <div class="left-side">
+        <img class="phone-img" src="images/phones.png">
+      </div>
+      <div class="right-side">
+        <div class="title-block">
+          <img class="instagram-img" src="images/brand.png">
+          <button class= "login">
+            <img class="home-img" src="images/home.png"> 
+            Log in
+          </button>
+          
+        </div>
+        <div class="central-text">
+          <h1 class="capture-share">
+            Capture and Share the World's Moments</h1>
 
-  Download on the App Store
-  Get it on Google Play
+          <p class="text-content">Instagram is a fast, beautiful and fun way to share your life with friends and family.</p>
 
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
+          <p class="text-content">Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that
+            easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.</p>
 
-  &copy; 2014 Instagram
+          <p class="text-content">Oh yeah, did we mention it's free?</p>
+        </div>
+        <div class="logos">
+          <img class="iphone-img" src="images/badge-iphone.png">
+          <img class="android-img" src="images/badge-android.png">
+        </div>
+      </div>
+    </div>
+    <div class="bottom-side">
+      <footer class="bottom-text">
+        <span class="bottom-element1">
+          ABOUT US SUPPORT BLOG PRESS API JOBS PRIVACY TERMS
```

Comment:
Better to put the "links" in a <ul> tag like this: 

```
  <footer>
    <ul>
      <li>
        <a href="#">About Us</a>
      </li>
      <li>
        <a href="#">Support</a>
      </li>
      <li>
        <a href="#">Blog</a>
      </li>
      <li>
        <a href="#">Press</a>
      </li>
      <li>
        <a href="#">API</a>
      </li>
      <li>
        <a href="#">Jobs</a>
      </li>
      <li>
        <a href="#">Privacy</a>
      </li>
      <li>
        <a href="#">Terms</a>
      </li>
      <li>&copy; 2014 Instagram</li>
    </ul>
  </footer>
```
In the css file you could customize this by adding

```
footer ul {
  clear: both;
  list-style: none;
  margin: 0 0 20px;
  padding: 0;
  text-align: center;
}

footer ul li {
  color: #999;
  display: inline;
  font-size: 11px;
  font-weight: bold;
  margin: 0 10px;
  text-transform: uppercase;
}

footer ul li a {
  color: #1c5380;
  text-decoration: none;
}

footer ul li a:hover {
  text-decoration: underline;
}
```